posts/2023-05-17-advanced-kotlin.adoc

```diff
@@ -0,0 +1,326 @@
+= Embrace Kotlin: Where to Get Started for Scala Developers
+AnastasiiaL
+v1.0, 2023-03-31
+:title: Exploring Advanced Kotlin Features: A Deep Dive for Scala Developers
+:imagesdir: ../media/2023-03-31-embrace-kotlin
+:lang: en
+:tags: [scala, kotlin]
+
+Recently I shared my experience with trying out Kotlin coming from a Scala background. I talked about the syntax and some concept similarities.
+In this blog post, I want to focus on Kotlin's more advanced features and explore how Scala developers can leverage their existing knowledge to make the most out of Kotlin.
+
+== Kotlin's Type System: A Closer Look for Scala Developers
+
+When first diving into Kotlin from a Scala background, one of the most noticeable differences is the simplicity of Kotlin's type system.
+While it's not as intricate as Scala's, it's robust and expressive, offering everything you need for most projects.
+In this section, we will delve deeper into Kotlin's type system, variance annotations, and type projections, and draw comparisons with Scala's type system.
```

Comment:
Since you've already used "and" after type projections, I would suggest replacing the second "and" with "as well as"